src/Compute/v2/Params.php

```diff
@@ -114,7 +114,7 @@ public function imageId(): array
     {
         return [
             'type'        => self::STRING_TYPE,
-            'required'    => true,
+            'required'    => false,
             'sentAs'      => 'imageRef',
             'description' => 'The unique ID of the image that this server will be based on',
```

Comment:
Please also update `description` to add clarity when `imageId` field is mandatory.

```The UUID of the image to use for your server instance. This is not required in case of boot from volume. In all other cases it is required and must be a valid UUID```